### DISCUSSION TITLE
Confirmation message on home page after submit on

### DISCUSSION THREAD

**Discussion by Gw3i** [neuefische/questions#111] [Web] [answered]
Hello everyone,
has someone an idea how I can display a confirmation text on the home page after submitting the edit form.
After submitting the edit form the user will be redirected back to the home page where the message would pop up. 

Here is a sketch:
<img width="623" alt="Screenshot 2022-08-25 at 15 12 40" src="https://user-images.githubusercontent.com/101993818/186674264-bbc3ccc6-de05-4d8e-8ee7-675ee145bda0.png">

Thanks in advance!

**Comment by mpagels** [comment, marked as answer]
Maybe you can store the information about, if a notification should be rendered in a global state. The default value is `false` (Nothing to notify) on the `Home`page. When you click `submit` and the submit was successful, before you redirect to the `Home` page, you change the `notify` state to `true` and the `Home` page is rendering the `notification` and after the notifcation is done, the `notify` state is changed back to `false`. Hope this helps and is understandable.

**Reply by Gw3i** [reply]
Great suggestion, thanks! Done
